zone/client_packet.cpp

```diff
@@ -7346,39 +7348,36 @@ void Client::Handle_OP_GroupInvite2(const EQApplicationPacket *app)
 					Invitee->CastToClient()->QueuePacket(outapp);
 					safe_delete(outapp);
 					return;
-				}
-				else
-				{
+				} else {
 					//The correct opcode, no reason to bother wasting time reconstructing the packet
 					Invitee->CastToClient()->QueuePacket(app);
 				}
-			}
-			else {
+			} else {
 				if (RuleB(Character, OnInviteReceiveAlreadyinGroupMessage)) {
 					if (!Invitee->CastToClient()->MercOnlyOrNoGroup()) {
 						Message(Chat::LightGray, "%s is already in another group.", Invitee->GetCleanName());
 					}
 				}
 			}
-		}
-		else if (Invitee->IsBot()) {
+		} else if (Invitee->IsBot()) {
 			Client* inviter = entity_list.GetClientByName(gis->inviter_name);
 			if (inviter && inviter->IsRaidGrouped() && !Invitee->HasRaid()) {
 				Bot::ProcessRaidInvite(Invitee->CastToBot(), inviter, true);
-			}
-			else if (!Invitee->HasRaid()) {
+			} else if (!Invitee->HasRaid()) {
 				Bot::ProcessBotGroupInvite(this, std::string(Invitee->GetName()));
 			} else {
 				MessageString(Chat::LightGray, ALREADY_IN_RAID, Invitee->GetCleanName());
 			}
 		}
-	}
-	else
-	{
-		auto pack = new ServerPacket(ServerOP_GroupInvite, sizeof(GroupInvite_Struct));
-		memcpy(pack->pBuffer, gis, sizeof(GroupInvite_Struct));
-		worldserver.SendPacket(pack);
-		safe_delete(pack);
+	} else {
+		if (RuleB(Character, GroupInvitesRequireTarget)) {
+			Message(Chat::White, "Invalid target!");
```

Comment:
Is this the message Live sent, or just one you put here for some sort of feedback?

Reply:
Feedback Only, nothing similar in eqstr_us that i could locate

Reply:
`12268 You must target a player first to invite to join your group.`

Reply:
Thanks, no idea how I missed that one

Reply:
12268 You must target a player or use /invite <name> to invite someone to your group.

That wont work for this case.

Reply:
Wasn't for the string code, was more for the text feedback

Reply:
Oh, I got you!